lib/methods/export.js

```diff
@@ -121,7 +158,7 @@ module.exports = self => {
             $in: draftIds
           }
         })
-        .relationships(includeRelationships)
+        .relationships(false)
```

Comment:
unneeded since we get related docs manually

Reply:
BC issue: someone may have overridden this method, so please check for 5 arguments and if there are five, accept them that way too, ignoring the redundant parameter.

Reply:
Riiiight BC, always forget that. Would be so good to have a public API, and internal methods that shouldn't be updated by users.